src/material/checkbox/checkbox.html

```diff
@@ -11,6 +11,7 @@
            [attr.aria-label]="ariaLabel || null"
            [attr.aria-labelledby]="ariaLabelledby"
            [attr.aria-describedby]="ariaDescribedby"
+           [attr.aria-checked]="indeterminate ? 'mixed' : checked"
```

Comment:
Should we do `null` if it's not indeterminate? I remember some time ago there being issues where screen readers would announce "checked" twice because of the `aria-checked`.

Reply:
Hm, I had that initially in my change (from a couple months ago) but I forgot why. Sounds good